source/timesoftfloat.c

```diff
@@ -2876,38 +2876,38 @@ enum { numInputs_extF80 = 32 };
 static
 const union { struct extFloat80M s; extFloat80_t f; }
     inputs_extF80[numInputs_extF80] = {
-        extF80Const( 0xC03F, 0xA9BE15A19C1E8B62 ),
```

Comment:
This is clearly unrelated to BF16.  What problem is being fixed?  Is the solution C99-compliant?

Reply:
It repairs compiler warning like `warning: missing braces around initializer`

Reply:
Sounds good if it doesn’t break anything.